krux_s3/s3.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+#
+# © 2015 Krux Digital, Inc.
+#
+
+# TODO: This is currently inside krux_manage_instance library.
+# However, consider breaking this into a separate library or add it to krux_boto library.
```

Comment:
I'm thinking you're doing this TODO right now.  Update in comments might be appropriate.